tests/unit/Codeception/ConfigurationTest.php

```diff
@@ -67,4 +67,83 @@ public function testDefaultCustomCommandConfig()
         $commandsConfig = $defaultConfig['extensions'];
         $this->assertArrayHasKey('commands', $commandsConfig);
     }
+
+    /**
+     * data provider for testGetRelativeDir
+     * 
+     * @return array(array(strings))
+     */
+    public function getRelativeDirTestData()
+    {
+        return [
+            // Unix style paths:
+                // projectDir() with & without trailing directory seperator: actual subdir
+                    ['/my/proj/path/some/file/in/my/proj.txt',           '/my/proj/path/',               '/',  'some/file/in/my/proj.txt'],
```

Comment:
I believe you were being sarcastic, but I disagree (so actually agreeing)... I think shorter is clearer... Just not sure how to pair this down better